site/content/includes/prometheus.yml

```diff
@@ -0,0 +1,33 @@
+global:
+  scrape_interval:     15s
+  evaluation_interval: 15s
+
+  external_labels:
+    monitor: 'codelab-monitor'
+
+remote_read:
```

Comment:
Do we want to include `remote_read` here since I think we only talk about querying m3query directly?

Reply:
Nevermind -- just saw where this is used.